Mathlib/Topology/Sober.lean

```diff
@@ -230,11 +230,22 @@ theorem quasiSober_of_open_cover (S : Set (Set α)) (hS : ∀ s : S, IsOpen (s :
   TopologicalSpace.IsOpenCover.quasiSober (U := fun s : S ↦ ⟨s, hS s⟩) <| by
     simpa [TopologicalSpace.IsOpenCover, ← SetLike.coe_set_eq, sUnion_eq_iUnion] using hS'
 
+/--
+Any R1 space is a quasi-sober space because any irreducible set is
+contained in the closure of a singleton.
+-/
+instance (priority := 100) R1Space.quasiSober [R1Space α] : QuasiSober α where
+  sober h hs := by
+    obtain ⟨x, hx⟩ := h.nonempty
+    use x
+    apply subset_antisymm
+    · rw [← hs.closure_eq]
+      exact closure_mono (singleton_subset_iff.mpr hx)
+    · exact isPreirreducible_iff_forall_subset_closure_singleton.mp h.isPreirreducible x hx
+
 /-- Any Hausdorff space is a quasi-sober space because any irreducible set is a singleton. -/
-instance (priority := 100) T2Space.quasiSober [T2Space α] : QuasiSober α where
-  sober h _ := by
-    obtain ⟨x, rfl⟩ := isIrreducible_iff_singleton.mp h
-    exact ⟨x, closure_singleton⟩
+instance (priority := 100) T2Space.quasiSober [T2Space α] : QuasiSober α :=
```

Comment:
Can you try commenting this line and benching the result? This you can achieve by writing bench! in the comment field on github. I suspect that this instance is basically useless.